cloud/pkg/common/messagelayer/context.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2022 The KubeEdge Authors.
```

Comment:
For the changed path files, do we should remain the year which means the time to implement them?

Reply:
the context.go in old directory do no have Copyright, and I have check that there are some other file do not have Copyright, I think we should add some check for the Copyright